styles/_mixins.scss

```diff
@@ -9,3 +9,17 @@
     @content;
   }
 }
+
```

Comment:
@samandar-boymurodov you should implement this feature in js
scroll bar should be shown only when user scrolls over scrollable element.
For example GitHub pull request details page

Reply:
Did not get it. Could you give another example?